packages/oruga/src/components/datepicker/Datepicker.vue

```diff
@@ -119,59 +119,83 @@
-                    :first-day-of-week="firstDayOfWeek"
-                    :rules-for-first-week="rulesForFirstWeek"
-                    :min-date="minDate"
-                    :max-date="maxDate"
-                    :focused="focusedDateData"
-                    :disabled="disabled"
-                    :unselectable-dates="unselectableDates"
-                    :unselectable-days-of-week="unselectableDaysOfWeek"
-                    :selectable-dates="selectableDates"
-                    :events="events"
-                    :indicators="indicators"
-                    :date-creator="dateCreator"
-                    :type-month="isTypeMonth"
-                    :nearby-month-days="nearbyMonthDays"
-                    :nearby-selectable-month-days="nearbySelectableMonthDays"
-                    :show-week-number="showWeekNumber"
-                    :week-number-clickable="weekNumberClickable"
-                    :range="range"
-                    :multiple="multiple"
-                    :table-class="tableClass"
-                    :table-head-class="tableHeadClass"
-                    :table-head-cell-class="tableHeadCellClass"
-                    :table-body-class="tableBodyClass"
-                    :table-row-class="tableRowClass"
-                    :table-cell-class="tableCellClass"
-                    :table-cell-selected-class="tableCellSelectedClass"
-                    :table-cell-first-selected-class="tableCellFirstSelectedClass"
-                    :table-cell-invisible-class="tableCellInvisibleClass"
-                    :table-cell-within-selected-class="tableCellWithinSelectedClass"
-                    :table-cell-last-selected-class="tableCellLastSelectedClass"
-                    :table-cell-first-hovered-class="tableCellFirstHoveredClass"
-                    :table-cell-within-hovered-class="tableCellWithinHoveredClass"
-                    :table-cell-last-hovered-class="tableCellLastHoveredClass"
-                    :table-cell-today-class="tableCellTodayClass"
-                    :table-cell-selectable-class="tableCellSelectableClass"
-                    :table-cell-unselectable-class="tableCellUnselectableClass"
-                    :table-cell-nearby-class="tableCellNearbyClass"
-                    :table-cell-events-class="tableCellEventsClass"
-                    :table-events-class="tableEventsClass"
-                    :table-event-variant-class="tableEventVariantClass"
-                    :table-event-class="tableEventClass"
-                    :table-event-indicators-class="tableEventIndicatorsClass"
-                    @range-start="date => $emit('range-start', date)"
-                    @range-end="date => $emit('range-end', date)"
-                    @close="togglePicker(false)"
-                    @update:focused="focusedDateData = $event" />
-
-
+                <div
+                    v-if="!isTypeMonth"
+                    class="datepicker-content"
```

Comment:
Why did you add this and the following lines?
You shouldn't touch the current use of `o-datepicker-table`

Reply:
I thought that the visibility of `o-datepicker-month` component or the `o-datapicker-table` one should be regulated by the value of `isTypeMonth` field. I used the same pattern used by `Datepicker` component of the `buefy` project

What the right thing to do ?

Reply:
Yes but we don't need that div, you can use `v-if` ok component itself or `template`

Reply:
done